### PR TITLE
2hcpjkj generator updates

### PR DESCRIPTION
**Pull request by shubham-shay-te**

### CODE REVIEW

File: website/src/components/MenuTitleChildren/MenuTitleChildren.jsx [hunk 1]
```diff
@@ -36,17 +35,22 @@ const HandleChildRenderEvents = (props) => {
 }
 
 const MenuTitleChildren = (props) => {
-    const children = []
-    useEffect(()=>{}, [props.items])
-    props.items.forEach((child, index) => {
-        children.push(
-           <HandleChildRenderEvents key={index} path={child.path} name={child.name} index={index} onClick={props.onClick} onDeleteClick={props.onDeleteClick} showDelete={props.showDelete}/>
+    const [children, setChildren] = useState([]);
```
Comment (shay-te) [inline website/src/components/MenuTitleChildren/MenuTitleChildren.jsx:38]: Does this component has no `props.children`? 
What component is it, Please can you attach a screenshot?

File: website/src/utils/PathsFactory.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    listPaths(path, yaml) {
+        const res = []
+        const list = this.#get(path, yaml) 
+        
+        if(!list){
+            return res
+        }
+        if (path === 'core_lib.entities') {
+            const entityRes = []
+            const connectionList = this.#get('core_lib.connections', yaml)
```
Comment (shay-te) [inline website/src/utils/PathsFactory.js:17]: `this.#get` ?

File: website/src/utils/PathsFactory.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+export default class Pathsfactory{
+    #get(path, yaml) {
```
Comment (shay-te) [inline website/src/utils/PathsFactory.js:2]: `#get` what is this syntax?

Comment (shubham-shay-te) [reply website/src/utils/PathsFactory.js:2]: just like `_` in python means its private to a class `#` is the syntax to do it in js

File: website/src/components/MenuTitleChildren/MenuTitleChildren.jsx (hunk 1 above)
Comment (shubham-shay-te) [reply website/src/components/MenuTitleChildren/MenuTitleChildren.jsx:38]: the component that renders the title and the list of children, on the right-hand side of the menu, for the` props.children` you had suggested to rename it to `items`

Comment (shubham-shay-te) [reply website/src/components/MenuTitleChildren/MenuTitleChildren.jsx:38]: image
this component this single title and children

**Review by shay-te — APPROVED** [review]

**Pull request merged**